submariner-operator/issue-149-node-placement.md

```diff
@@ -0,0 +1,96 @@
+# Node Placement
+
+[Add support for nodeSelector and toleration scheduling for submariner pods](https://github.com/submariner-io/enhancements/issues/149)
+
+## Summary
+
+Submariner deploys many different agents as part of its core deployment. Gateway, Globalnet Agent and
+Metrics Proxy are deployed on designated gateway nodes, while Route Aents are deployed on all nodes. There is no way
+for users to control or limit where rest of the agent pods are deployed. Some customers want to run these agents on
+specific nodes for billing or networking reasons. Users can't add tolerations directly to these agents coz they'll be
+overwritten by the operator. So, users require an easily configurable way to specify such configurations.
+
+## Proposal
+
+This enhancement proposes adding a new field to `Submariner` and `ServiceDiscovery` for users to configure node
+scheduling. This configuration will then be applied to pods of following submariner agents:
+
+* submariner-operator
```

Comment:
I believe you can add the network-plugin-syncer pod to the list, IIUC it doesnt need a pecific place to run but I might be wrong

Reply:
Good catch, missed it.